ProjectCars/CarCRUD.cs

```diff
@@ -1,27 +1,25 @@
-﻿using System;
-using System.Collections.Generic;
+﻿using System.Collections.Generic;
+using System.IO;
 using System.Linq;
-using System.Text;
-using System.Text.Json;
-using System.Threading.Tasks;
+using Newtonsoft.Json;
 
 namespace ProjectCars
 {
-     public class CarCRUD
+    public class CarCRUD
     {
-        //private static string path = @"C:\Json Store\Cars.json";
+        private static string _path = @"C:\Json Store\Cars.json"; //I´m going to change this
```

Comment:
It would be nice if you can make this somehow configurable by using `appsettings.json`.